xdrip-get-entries.sh

```diff
@@ -949,11 +949,13 @@ function check_variation()
     return
   fi
   variation=$(bc <<< "($filtered - $unfiltered) * 100 / $filtered")
-  if [ $(bc <<< "$variation > 10") -eq 1 -o $(bc <<< "$variation < -10") -eq 1 ]; then
-    log "would not allow meter calibration - filtered/unfiltered variation of $variation exceeds 10%"
-    meterbg=""
-  else
-    log "filtered/unfiltered variation ok for meter calibration, $variation"
+  if [ $found_meterbg == false ]; then
```

Comment:
why check found_meterbg==false? doesn't that mean there is no meterbg? so checking variation and resetting meterbg is superfluous then...

Reply:
Correct, I noticed my log file had these logs in it every Logger loop even though meterbg wasn't present. It was confusing and unneccessary so checking if meterbg actually exists before the logs (and potential removal of meterbg if variation threshold is met) seems to be cleaner.